src/providers/UserInterestProvider.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { testDataHandler } from '../utils/Utils';
+
+export default class UserIntrestProvider {
+  userInterest() {
+    const ENTITYINFOEXAMPLES = testDataHandler('content', 'CONTENT');
+    if (ENTITYINFOEXAMPLES.hasOwnProperty(process.env.entityInfoKey)) {
```

Comment:
the response of **Content.requestUserInterest()** is expected to be in format of **EntityDetails**, there is already data in this **EntityDetails** format for federeated data, just using of same data, from FCS before operation will send the keyname of which to send back as response, that value will be send back from here

Reply:
Is the work from FCS side is done?

Reply:
yes